test/models/movie.test.js

```diff
@@ -0,0 +1,22 @@
+'use strict';
+
+const Movie = require('../../lib/models/movie');
+
+describe('movie model', () => {
+
+  describe('serialize', () => {
+
+    it('includes all of the necessary fields', () => {
```

Comment:
If there's more complicated logic in the serialize function, it can be good practice to also assert the the values of certain fields. In this case, this is probably sufficient. 👍